examples/demo10.py

```diff
@@ -166,14 +171,21 @@ def main(argv):
     # applied.
     galsim.random.permute(rng, ix_list, iy_list)
 
-    # Now have the PowerSpectrum object build a grid of shear values for us to use.
-    grid_g1, grid_g2 = ps.buildGrid(grid_spacing=stamp_size*pixel_scale, ngrid=n_tiles,
-                                    rng=rng)
+    # Initialize the OutputCatalog for the truth values
```

Comment:
Should the docstring for the demo mention that this demo shows the use of the OutputCatalog functionality?